tools/parse_baselines/providers/snapshot_restore.py

```diff
@@ -33,8 +33,14 @@ def __init__(self, data_provider: Iterator):
     def calculate_baseline(self, data: List[float]) -> dict:
         """Return the target and delta values, given a list of data points."""
         avg = statistics.mean(data)
-        stddev = statistics.stdev(data)
+        min_ = min(data)
+        max_ = max(data)
+        print(len(data))
+        min_delta = 100 * abs(avg - min_) / avg
+        max_delta = 100 * abs(avg - max_) / avg
+        delta = max(max_delta, min_delta)
+
         return {
             "target": round(avg, 3),
-            "delta_percentage": math.ceil(3 * stddev / avg * 100) + DELTA_EXTRA_MARGIN,
+            "delta_percentage": math.ceil(delta) + DELTA_EXTRA_MARGIN,
```

Comment:
Doesn't this logic mean that we should re-run performance baselines for everything?

Also, do you see any value in keeping in the data format both the delta as in (max - min) and the standard deviation? Even though we will only use the delta for signaling problems with performance the standard deviation could be useful information when we plot data.

Reply:
Not necessarily since the current logic should include the old values (actually this is a more relaxed way to compute values -> what it does is that it will include the entire range covered by a number of build).